https://bugs.gentoo.org/938229

From 1fb93b68d15c7c039f04d0145ad47fb3102ec359 Mon Sep 17 00:00:00 2001
From: Sergei Trofimovich <slyich@gmail.com>
Date: Sat, 24 Aug 2024 22:21:47 +0100
Subject: [PATCH] filesystem.hpp: include missing <cstdint>

Without the change build fails on upcoming `gcc-15` as:

    In file included from src/desktop/paths.cpp:20:
    src/filesystem.hpp:232:13: error: 'uint8_t' was not declared in this scope
      232 | std::vector<uint8_t> read_file_binary(const std::string& fname);
          |             ^~~~~~~
---
 src/filesystem.hpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/filesystem.hpp b/src/filesystem.hpp
index 38bf6952b452c..3cdb4b6ae7120 100644
--- a/src/filesystem.hpp
+++ b/src/filesystem.hpp
@@ -21,6 +21,7 @@
 #pragma once
 
 #include <ctime>
+#include <cstdint>
 #include <fstream>
 #include <iosfwd>
 #include <memory>
